From f2a17cf67b93b9eed9a08713f667cd0a9b71dd23 Mon Sep 17 00:00:00 2001
From: ShevT <shev_t@mail.ru>
Date: Thu, 26 Jul 2018 11:47:47 +0300
Subject: [PATCH 09/11] healthd: BatteryMonitor: Fix compiler warning

system/core/healthd/BatteryMonitor.cpp:248:19: note: initialize the variable 'i' to silence this warning
    unsigned int i;
                  ^
                   = 0
1 warning generated.

Change-Id: Ib2fa0061d657fd93871ca8a9056d173d5b7ab58d
---
 healthd/BatteryMonitor.cpp | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/healthd/BatteryMonitor.cpp b/healthd/BatteryMonitor.cpp
index b32343850..9c7289542 100644
--- a/healthd/BatteryMonitor.cpp
+++ b/healthd/BatteryMonitor.cpp
@@ -243,7 +243,6 @@ bool BatteryMonitor::update(void) {
     if (readFromFile(mHealthdConfig->batteryTechnologyPath, &buf) > 0)
         props.batteryTechnology = String8(buf.c_str());
 
-    unsigned int i;
     double MaxPower = 0;
 
     // Rescan for the available charger types
@@ -280,7 +279,7 @@ bool BatteryMonitor::update(void) {
         }
     }
 
-    for (i = 0; i < mChargerNames.size(); i++) {
+    for (unsigned int i = 0; i < mChargerNames.size(); i++) {
         String8 path;
         path.appendFormat("%s/%s/online", POWER_SUPPLY_SYSFS_PATH,
                           mChargerNames[i].string());
-- 
2.17.1

